timescale/timescale.go

```diff
@@ -25,7 +25,7 @@ func New(db *sqlx.DB) callhome.TelemetryRepo {
 func (r repo) RetrieveAll(ctx context.Context, pm callhome.PageMetadata) (callhome.TelemetryPage, error) {
 	q := `
 	WITH aggregated_data AS (
-		SELECT ip_address, ARRAY_AGG(service) AS services
+		SELECT ip_address, ARRAY_AGG(DISTINCT service) AS services
```

Comment:
Why do we need to use distinct, since each svc name should be unique per IP?

Reply:
There was a repetition in the services array `["thing", "thing"]` from the aggregated records